### PR TITLE
bindAsObject/bindAsArray example

### PR DESCRIPTION
**Pull request by mattiasbpersson**
bindasobject

### CODE REVIEW

**Comment by mattiasbpersson** [comment]
bindasarray

File: src/components/firebase_examples/FruitOnFire.vue [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    // Bind fruit variable to fruit_data attribute in firebase
+    this.$bindAsObject('fruit', db.ref('fruity_data'))
+  },
+  methods: {
+    save (event) {
+      console.log(this.fruit)
+      // If nothing exists in firebase it will create an object with a .key and .value attribute.
+      // set-method doesn't approve
+      delete this.fruit['.key']
+      delete this.fruit['.value']
```
Comment (ruaridhwatt) [inline src/components/firebase_examples/FruitOnFire.vue:45]: I don't understand! Why do we need to delete these?

**Review by ruaridhwatt — COMMENTED** [review]
I'm not sure if Vuefire is clearer/easier than doing it the standard firebase js way.
https://firebase.google.com/docs/database/web/read-and-write?authuser=1
What are your thoughts?

**Comment by ruaridhwatt** [comment]
Perfect examples. Just wonder if it would be better to start with firebase and then mix in vuefire after an example or two.

**Comment by ruaridhwatt** [comment]
I did some more without vuefire and I see now that we will not have time on friday to go through firebase js sdk and then vuefire. In fact, I now see that vuefire is very good at updating our data objects on firebase events. it reduces the cognitive load and will be easier to be effective as a mob using vuefire.

tldr;
We hop over raw firebase js sdk and go straight into vuefire! (Just as you have done :) ) Nice work!

File: src/components/firebase_examples/FruitOnFire.vue (hunk 1 above)
Comment (ruaridhwatt) [reply src/components/firebase_examples/FruitOnFire.vue:45]: Can we do it another way so we do not have to do this?

**Comment by mattiasbpersson** [comment]
bindasarray_dataobjects
